source/manual/dns.html.md

```diff
@@ -15,76 +15,100 @@ GOV.UK is responsible for managing several DNS zones, spanning a number of `*.go
 gds aws govuk-production-poweruser -- aws route53 list-hosted-zones | grep Name
 ```
 
-## Records for GOV.UK systems
+## Overview of GOV.UK domains
 
-We use a few domains:
+GOV.UK is responsible for more than just the `www.gov.uk` domain.
+The [GOV.UK proposition](https://www.gov.uk/government/publications/govuk-proposition/govuk-proposition#what-the-govuk-proposition-covers) lists the domains we're responsible for. These are:
 
-- `alphagov.co.uk` is the old domain name GOV.UK publishing used to live on.
-  We maintain records which point to Bouncer so that these URLs redirect.
-- `publishing.service.gov.uk` and `govuk.service.gov.uk` are where GOV.UK lives.
+- www.gov.uk
+- service.gov.uk
+- data.gov.uk
+- blog.gov.uk
+- campaign.gov.uk
+- independent.gov.uk
+- api.gov.uk
 
-## DNS for `*.service.gov.uk` domains
+There are lots of other `*.gov.uk` domains, such as `cityoflondon.gov.uk`, which are managed by Cabinet Office.
 
-GOV.UK Technical 2nd Line are responsible for delegating DNS to other government services.
-Note that we __do not__ manage any other DNS records: if you get a request concerning anything other than `NS` records, it should be rejected.
+In theory, the GOV.UK proposition domains should all be managed by GDS (who use [Jisc](https://www.jisc.ac.uk/): a non-profit that provides networking to UK education and government). In practice, a couple of GOV.UK proposition domains are managed by Cabinet Office, and a couple of non-proposition domains are in the Government Digital Service Jisc account. [This is being looked at](https://trello.com/c/qNpyVaC5/3228-consolidate-co-vs-non-co-domains-in-govuks-jisc-account) by Platform Security & Reliability.
```

Comment:
Wondering if we need some sort of 'terminology' section to describe Cabinet Office vs Government Digital Service vs GOV.UK?

The doc pretty much refers to GOV.UK and Government Digital Service interchangeably, though I've tried to use the right one in context, i.e. "Government Digital Service" _is_ the name of our Jisc account, and "GOV.UK" is the department linked to the proposition, etc.